### PR TITLE
L1t stage1 unpacker hlt 76x

### PR DESCRIPTION
**Pull request by mulhearn**
This PR brings the L1T packer/unpacker tool in line with (not yet available) AMC13 firmware. It is necessary for data taking with the Stage 1 trigger.  Because of a backwards incompatibility between the actual firmware and the offline anticipation of the firmware, additional customization will be needed to unpack L1 payload from RAW data produced via MC under 74X, if such capability is needed.

This PR (will) include the customization for HLT promised in PR #9981.

### CODE REVIEW

**Comment by mulhearn** [comment]
please test

**Comment by mulhearn** [comment]
please test

**Comment by mulhearn** [comment]
+1

**Comment by mulhearn** [comment]
@Martin-Grunewald:  I believe this handles the HLT configuration changes correctly now.  Can you confirm?

**Comment by Martin-Grunewald** [comment]
+1

**Comment by davidlange6** [comment]
@mulhearn - Could you summarize the use of each of the data files?

**Comment by mulhearn** [comment]
(asking)

**Comment by mulhearn** [comment]
These are all for unit testing.  We'd therefore like them in the release eventually, but they aren't currently needed for any offline workflows...

**Comment by mulhearn** [comment]
@Martin-Grunewald:  Not sure I fully understand the HLT config stuff as I prepare the 74X tags.... do you use the same PR to label the customization fragments in all of e.g. 74X/75X/76X even though there is a different PR in each case?

**Comment by mulhearn** [comment]
(typo "as I prepare the 74X PR")

**Comment by Martin-Grunewald** [comment]
It can not be the same PR as each PR is for only ONE CMSSW release series.
I do not use PRs to label customization fragments !!!
Maybe I do not understand your question???

**Comment by mulhearn** [comment]
I'm talking about e.g. 
   def customiseFor10087(process):

**Comment by Martin-Grunewald** [comment]
Ah, in that specific file - but see there it gets only executed if we are in 75 or later
(the CMSSW version is checked). Usually we keep this file the same between
releases... but I think you latest change there is only in 76X anyway.
